game/script-events.rpy

```diff
@@ -1270,5 +1271,281 @@ label event_wintendo_twitch_game_over:
     n 1fsqajl "Well?{w=0.5}{nw}"
     extend 1fcspolesi " Get to it then,{w=0.2} [player]!"
     n 1fsqsml " Ehehe."
-    
+
+    return
+
+# Natsuki shares tips how to stay warm at chilly days
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._event_database,
+            label="event_warm_package",
+            unlocked=True,
+            conditional="jn_utils.get_total_gameplay_days() >= 21 and persistent.jn_custom_outfits_unlocked",
+            affinity_range=(jn_affinity.HAPPY, None)
+        ),
+        topic_group=TOPIC_TYPE_EVENT
+    )
+
+label event_warm_package:
+    python:
+        jn_globals.force_quit_enabled = False
+        teddy_cardigan_outfit = jn_outfits.get_outfit("jn_teddy_cardigan_outfit")
+        teddy_cardigan_outfit.unlock()
+        Natsuki.setOutfit(teddy_cardigan_outfit)
+
+    # 0 = rain, 1 = snow, 2 = default
+    $ test_int_condition = 1
+
+    if test_int_condition == 0:
+    #if jn_atmosphere.isCurrentWeatherRain() or jn_atmosphere.isCurrentWeatherThunder():
+        n "..."
+        n "Uuuuuuu-!"
+        n "You've {w=0.2}{i}got{/i}{w=0.2} to be kidding me."
+        n "Rain?!{w=0.75} {i}Again?!{/i}"
+        n "It's always freezing here when it does that!{w=1} I don't even {i}have{/i} a radiator to turn on!"
+        $ jnPause(3)
+        n "Ugh..."
+        n "You know what?{w=0.75} Screw this!"
+        play audio chair_out
+        n "Someone {i}had{/i} to have left a coat or...{w=0.5} {i}something{/i} lying around..."
+        $ jnPause(3)
+
+        play audio clothing_ruffle
+        $ jnPause(2)
+        play audio clothing_ruffle
+
+        n "..."
+        n "Jeez..."
+        n "How is this not enough?{w=1} I'm {i}still{/i} freezing my butt off!"
+
+        $ jnPause(3)
+        play audio gift_slide
+        $ jnPause(1)
+        play audio gift_open
+
+        n "Uuuuu..."
+        n "You'd think the {i}star{/i} of the debate club would have at least {i}tried{/i} to talk our way into a warm clubroom."
+        n "I can barely feel my toes..."
+
+        $ jnPause(2)
+        play audio chair_out
```

Comment:
We should remove these two lines, as the chair sound doesn't make sense when she's in the closet!